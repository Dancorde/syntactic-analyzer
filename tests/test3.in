program lalg;
{numeros grandes}
var a: integer;
begin
readd(431.45534534453, a, 139821748934623);
end.