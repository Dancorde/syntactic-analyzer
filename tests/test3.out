program - program
lalg - id
; - ;
var - var
a - id
: - :
integer - integer
; - ;
begin - begin
readd - id
( - (
431.45534534453 - erro - numero real muito grande
, - ,
a - id
, - ,
139821748934623 - erro - numero inteiro muito grande
) - )
; - ;
end - end
. - .
